### PR TITLE
at.cpp: Added NEW MACHINES Ericsson/Nokia WS286

### PR DESCRIPTION
**Pull request by JoakimLarsson**

### CODE REVIEW

File: src/mame/drivers/at.cpp [hunk 1]
```diff
@@ -1286,6 +1306,20 @@ ROM_START( comportiii )
...
+	ROM_LOAD16_BYTE( "RYS_103_1002_R8A_3C00_IC-POS_71.BIN", 0x18000, 0x4000, CRC(af179e56) SHA1(58b1df46d6e68eef472a0529cb9317abaf17880f)) // Last ROM set and has Nokia
+	ROM_LOAD16_BYTE( "RYS_103_1003_R8A_8600_IC-POS_69.BIN", 0x18001, 0x4000, CRC(555502cb) SHA1(1977fe54b69c5e52731bf3eb8bdabe777aac014b)) // copyright patched in both roms
+ROM_END
+
+// Nokia Data WS286
+ROM_START(nws286 ) // Computer is grey with Nokia logo.
+	ROM_REGION(0x20000,"bios", 0)
+	ROM_LOAD16_BYTE( "RYS_103_1002_R8A_3C00_IC-POS_71.BIN", 0x18000, 0x4000, NO_DUMP)
+	ROM_LOAD16_BYTE( "RYS_103_1003_R8A_8600_IC-POS_69.BIN", 0x18001, 0x4000, NO_DUMP)
+ROM_END
```
Comment (DopefishJustin) [inline src/mame/drivers/at.cpp]: If there are no dumps at all for this machine then it should not be added as a driver. Add the information in comments instead.

Comment (JoakimLarsson) [reply src/mame/drivers/at.cpp]: Ok, I am looking for a newer rom, but I can't tell when it pops up, I would think the ews286 works but I want to get it from an actual Nokia branded machine to be authentic, I'll comment it out for now.

Comment (JoakimLarsson) [reply src/mame/drivers/at.cpp]: And I tried to use ews286 as parent but that didn't validate

**Comment by rb6502** [comment]
In what way is the ROM not authentic?

**Comment by DavidHaywood** [comment]
well since it is listed as NO_DUMP for all roms, I guess it isn't authentic because it doesn't exist?

**Comment by JoakimLarsson** [comment]
Well, the machine it is dumped from has Ericsson logos but a Nokia keyboard and the roms has Nokia copyright. The roms are authentic but I am not sure for which machine, it could be that someone upgraded the roms from a Nokia machine or the whole motherboard in which case it is a Nokia rom. Or it could be a Ericsson branded delivery after Nokia Data acquired Ericsson Information Systems in which case it is an Ericsson rom with just the copyright patched.

**Pull request merged**